tbx/project_styleguide/templates/patterns/pages/blog/blog_detail.html

```diff
@@ -54,7 +54,8 @@
 
             <ul class="grid__related-posts streamfield__related-posts">
                 {% for post in page.related_blog_posts %}
-                    {% include "patterns/molecules/listing/listing--avatar.html" with title=post.title name=post.author.name job_title=post.author.role link=post.url date=post.date reading_time=post.read_time tags=post.tags avatar=post.author.image tag_link_base=blog_index_url %}
+                    {% pageurl post as post_url %}
```

Comment:
The pageurl mock needs updating in the pattern library here to avoid a `/` appearing:

![Screenshot 2024-04-30 at 12 17 06](https://github.com/torchbox/torchbox.com/assets/771869/f681edbf-61b4-4dd1-a8c8-59c374081db0)

Reply:
We're also now missing the author details - compare with the `main` branch:

![Screenshot 2024-04-30 at 12 18 11](https://github.com/torchbox/torchbox.com/assets/771869/da98863b-b6b8-4300-a1c2-608a6471d49c)